core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -62,8 +62,16 @@ class QualificationAppInfo(
   var clusterTags: String = ""
   var clusterTagClusterId: String = ""
   var clusterTagClusterName: String = ""
+  var clusterInfo: Option[ClusterInfo] = None
```

Comment:
Currently cluster detection is implemented only for Q tool. 

We can move to `AppBase` with some refactoring. `Platform` must be made available to `AppBase`

Reply:
No worries.
I believe this refactor could be part of the Tuning revamp since we will need the HW configurations to suggest different HW shape for tuning.
I will work on that next Spring.

Reply:
Thanks @amahussein.